### ISSUE TITLE
Create custom biome and IDs

### ISSUE THREAD

**Issue by MRLexanayseR** [superckl/BiomeTweaker#241]
Dear developer. I ask for your help. Is it possible to change the ID in my created biomes? ID 41 is set by default, but I need another, how to implement this? Thank you in advance.

I am applying an example.
https://ibb.co/swmyhdH

**Comment by superckl** [comment]
Why. IDs do not matter. You should only be using resource locations.

**Comment by MRLexanayseR** [comment]
The problem is that I create a map through **WorldPainter** and create my biomes through **your mod**. But, in World Painter, their biomes are placed starting from **ID 51**, and in your mod, they start from **ID 41**. I can’t change it anywhere. The program does not change exactly. Therefore, I asked you whether it is possible to change the biome ID to your own, say 51.

Comparison:
https://ibb.co/48D1Ryr
https://ibb.co/swmyhdH

While I found only one way out, it is crazy.
Through your mod, through a script, I created 9 empty biomes. 10 biome, it turned out to be 51 biome IDs. And in the game on the map everything works. But running the extra 24 lines of code to achieve 51 biome IDs is very stupid.

An example of what I did:
https://ibb.co/ZzKYQXc
https://ibb.co/k5dYVns
